concrete/src/Support/i18n.php

```diff
@@ -8,3 +8,11 @@
 
 // To translate the value of the VERSION_INITIAL_COMMENT constant
 t('Initial Version');
+
+// To translate the names of the autentication types
+// @see \Concrete\Core\Package\StartingPointPackage::add_users
+tc('AuthenticationType', 'Standard');
```

Comment:
Shouldn't these be handled dynamically as with other objects that have translatable names? Not sure what the current convention is, though.